static/js/popovers.js

```diff
@@ -926,8 +1054,28 @@ exports.register_click_handlers = function () {
     });
 
     $('body').on('click', '.flatpickr-confirm', function (e) {
-        var datestr = $(".remind.custom")[0].value;
-        reminder_click_handler(datestr, e);
+        var datestr;
+        if (send_later_popover_elem !== undefined) {
```

Comment:
Is there no better way to tell whether the flatpickr element is from the reminder or send-later feature?  If that's the case, we should write a comment explaining that; otherwise, it'd be cleaner to e.g. check a class on the actual flatpickr element that indicates what widget it's for than checking which popover is open.